src/client/java/minicraft/saveload/Load.java

```diff
@@ -1129,7 +1129,7 @@ private static Entity getEntity(String string, int mobLevel) {
 			case "Workbench": return new Crafter(Crafter.Type.Workbench);
 			case "Chest": return new Chest();
 			case "DeathChest": return new DeathChest();
-			case "DungeonChest": return new DungeonChest(false);
+			case "DungeonChest": return new DungeonChest(null);
```

Comment:
These will probably cause nullpointerexceptions.

Reply:
`null` case is handled within `DungeonChest`.

Reply:
Alright, I see what you mean. But do we want the chests to be empty in those cases. And should the populate inventory function even be called from the DungeonChest class?

Reply:
In `Load`, this method fetches the initial state (empty state this case) for further entity loading, although actually this is quite badly designed.

Reply:
Will empty locked chest populate?

Reply:
If it is loaded from saves, the chests are never populated.

Reply:
Also, the initial state of "locked" does not affect the final state of the chest. The saved value of "locked" should always be loaded from the save after the initial construction of the instance.

Reply:
Locked dungeon chests that spawn in structures method are empty while those that are spawned via the other method are populated.

Reply:
![image](https://github.com/MinicraftPlus/minicraft-plus-revived/assets/74168521/f15583e9-b59c-4c49-bf51-faf591cf0f17)

Reply:
I hope I have not misunderstood.

Reply:
I’ll do more testing, just fix the conflicts too please.